client/src/pages/NrMap.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect, useRef } from "react";
+import React, { useState, useEffect, useRef, useCallback } from "react";
```

Comment:
This seems out of scope for this PR ? 

@agmaal1 was working on this - can you double check if this is fine please?

Reply:
@anniegiang I can revert it, only reason why I did that is when I ran the linter this came up as an error, `useCallback` is being used without an import, same with with render on the functional component

Reply:
Ah ok, maybe we would fix that error in other PR.